docs/dyndns/update-api.rst

```diff
@@ -20,23 +20,35 @@ connections.
 
 Authentication
 **************
-You can authenticate your client in several ways:
+You can authenticate your client in several ways. If authentication fails, the
+API will return a ``401 Unauthorized`` status code.
+
+Preferred method: HTTP Basic Authentication
```

Comment:
to be very obvious, maybe call this "HTTP Basic Token Authentication"

Reply:
done similarly